test/dynamo/test_higher_order_ops.py

```diff
@@ -1037,25 +1037,13 @@ def false_fn(x):
 
         mod_for_compile = torch.compile(Foo(), backend=cnt, dynamic=True)
         mod_for_eager = Foo()
-
-        actual = mod_for_compile(torch.ones(6, 4))
         ref = mod_for_eager(torch.ones(6, 4))
-        self.assertEqual(actual, ref)
-
-        actual = mod_for_compile(torch.ones(3, 4))
-        ref = mod_for_eager(torch.ones(3, 4))
-        self.assertEqual(actual, ref)
 
-        self.assertExpectedInline(
-            backend.graphs[0].code.strip(),
-            """\
-def forward(self, s0 : torch.SymInt, s1 : torch.SymInt, L_x_ : torch.Tensor):
-    l_x_ = L_x_
-    size = l_x_.size();  l_x_ = None
-    getitem = size[0];  size = None
-    gt = getitem > 4;  getitem = None
-    return (gt,)""",
-        )
+        with self.assertRaisesRegex(
```

Comment:
Btw, does this work with surpress_errors = True?

Reply:
yes, GRAPH_BREAK_IN_CONTROL_FLOW check precedes the [suppress_error check](https://github.com/pytorch/pytorch/blob/main/torch/_dynamo/convert_frame.py#L658)

Reply:
Can you add a test that turns suppress_errors=True and check this?